### DISCUSSION TITLE
Adding a feature to label in chat when items depawn?

### DISCUSSION THREAD

**Discussion by BrndnWlsh** [gnembon/fabric-carpet#1114] [Ideas]
not sure of how difficult this would be, but it would be very useful in everything from farming to mining ores in a pillar with fortune. if this is unfeasible, I think a similar feature related to item despawns could be very useful.

**Comment by Fallen-Breath** [comment]
`/log item despawn` or lifetime tracker in carpet tis addition might be what you need
